psyslog.js

```diff
@@ -6,15 +6,41 @@ const pump = require('pump')
 const split2 = require('split2')
 const parseJson = require('fast-json-parse')
 
+let options = {
```

Comment:
Why did this change names and become a `let` instead of a `const`?

Reply:
Because it is reassigned below. const should only used for constant values. Chrome for example doesn't allow reassigning new values to const variables.

Reply:
That's the point of using it. The defaults object should not be mutated. As I mentioned in another review comment, that was a bug.

Reply:
I changed it back to const.